src/TestHelper/RequestObjectTestHelper.php

```diff
@@ -144,15 +144,19 @@ public function getRequestProperties(RequestObjectInterface $object): array
         $methodsToCheck = \array_filter(
             \array_diff($instanceMethods, $interfaceMethods),
             static function (string $method): bool {
-                return \strncmp($method, 'get', 3) === 0;
+                return \strncmp($method, 'get', 3) === 0 ||
+                    \strncmp($method, 'is', 2) === 0;
             }
         );
 
         $actual = [];
-        foreach ($methodsToCheck as $method) {
-            $property = \lcfirst(\substr($method, 3));
+        \asort($methodsToCheck);
```

Comment:
If we're going to sort alphabetically we need a comment to explain why

Reply:
Added comment

Reply:
Comment is not good English.

Reply:
Rephrased the comment!